compute/etc/neon_collector_autoscaling.jsonnet

```diff
@@ -0,0 +1,67 @@
+{
+  collector_name: 'neon_collector_autoscaling',
+  metrics: [
+    {
+      metric_name: 'lfc_misses',
+      type: 'gauge',
+      help: 'lfc_misses',
+      key_labels: null,
+      values: [
+        'lfc_misses',
+      ],
+      query: "select lfc_value as lfc_misses from neon.neon_lfc_stats where lfc_key='file_cache_misses';\n",
```

Comment:
This still directly uses the queries — is that intentional?

Reply:
No, good catch.